sycl/test-e2e/ESIMD/accessor_global.cpp

```diff
@@ -0,0 +1,121 @@
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// UNSUPPORTED: gpu-intel-gen9 && windows
+//
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies usage of accessor methods operator[] and get_pointer().
+
+#include "esimd_test_utils.hpp"
+
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+#include <iostream>
+
+using namespace sycl;
+using namespace sycl::ext::intel::esimd;
+
+constexpr int VL = 16;
+
+template <typename T, bool TestSubscript>
+bool test(queue Q, uint32_t LocalRange, uint32_t GlobalRange) {
+  std::cout << "Running case: T=" << esimd_test::type_name<T>()
+            << ", TestSubscript=" << TestSubscript << std::endl;
+
+  T *Tmp = new T[GlobalRange * VL];
+  T *Out = malloc_shared<T>(GlobalRange * VL, Q);
+  for (int I = 0; I < GlobalRange * VL; I++)
+    Out[I] = -1;
+
+  try {
+    nd_range<1> NDRange{range<1>{GlobalRange}, range<1>{LocalRange}};
+    buffer<T, 1> TmpBuf(GlobalRange * VL);
+    Q.submit([&](handler &CGH) {
+       auto TmpAcc = TmpBuf.template get_access<access::mode::read_write>(CGH);
```

Comment:
```suggestion
       accessor TmpAcc{TmpBuf, CGH};
```

Reply:
Fixed in 135ed67d5a82d2edf5535e7262bbc15f4d3e7f5d